Resources/Prototypes/Entities/Objects/Tools/welders.yml

```diff
@@ -39,6 +38,12 @@
       collection: Welder
     qualities: Welding
   - type: Welder
+    litDamageModifier:
+      coefficients:
+        Blunt: 0 # you're burning them, not bashing them.
+    litDamageBonus:
+      types:
+        Heat: 10 # add 10 heat damage.
```

Comment:
why cant the damagebonus just have `Blunt: -10`

Reply:
Currently that should achieve the same thing, but I think its better this way just in case anyone ever tweaks the welder melee damage without updating this. E.g., if they reduced it from 10, then the -10 blunt damage would start healing the target a bit.

Reply:
I've switched to just making it use Blunt: -10.